test/performance/odp_dma_perf.c

```diff
@@ -130,6 +141,7 @@ static void print_usage(void)
 	       "    odp_dma_perf\n"
 	       "    odp_dma_perf -t 0 -g 1 -i 2\n"
 	       "    odp_dma_perf -t 1 -g 1 -i 4 -m 0,0,0,0\n"
+	       "    odp_dma_perf -t 1 -g 1 -i 7 -m 0,0,0,0,0,0,1 -T 1 -r 1000 -s 2048\n"
```

Comment:
It would be good to run the application with both data types in the CI. `test/performance/odp_sched_perf_run.sh` is one example of how this can be done.

Reply:
Sure. Could be a separate PR though.